lms/djangoapps/onboarding/forms.py

```diff
@@ -371,13 +372,11 @@ def save(self, request, commit=True):
         if not is_currently_unemployed and organization_name:
             organization_to_assign = Organization.objects.filter(label__iexact=organization_name).first()
             if not organization_to_assign:
-                organization_to_assign = Organization.objects.create(label=organization_name)
+                organization_to_assign, organization_created = Organization.objects.create(label=organization_name), True
 
             prev_org = user_info_survey.organization
             user_info_survey.organization = organization_to_assign
-
-            if organization_to_assign.users_count() == 0:
-                user_info_survey.is_first_learner = True
+            user_info_survey.is_first_learner = (organization_to_assign.users_count() == 0 and organization_created)
```

Comment:
I think according to current logic, we don't need to check users_count here.
"As according to discussion user will be first_learner only if he created the organization else he will not be first_learner"